Cocktails9/Cocktails9/Model/FilterModel.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  FilterModel.swift
```

Comment:
Naming convention for models is without 'model', just 'Filter', 'User', 'Drink'...